src/RedirectStorageSchema.php

```diff
@@ -27,9 +27,49 @@ protected function getEntitySchema(ContentEntityTypeInterface $entity_type, $res
     ];
     $schema['redirect']['indexes'] += [
       // Limit length to 191.
-      'source_language' => [['redirect_source__path', 191],'language'],
+      'source_language' => [['redirect_source__path', 191], 'language'],
     ];
-
+    $schema['redirect_error'] = array(
+      'description' => 'Stores information on redirect errors.',
+      'fields' => array(
+        'reid' => array(
```

Comment:
Unsure what is the storage primary id pattern now. Are we still prefixing ids? Or should we just use id?